pkg/expr/func_time.go

```diff
@@ -195,3 +196,27 @@ func strfTime(base interface{}, f string) (string, error) {
 	o, _ := strftime.Format(f, *t, strftime.WithMilliseconds('b'))
 	return o, nil
 }
+
+// sub returns difference between two date into milliseconds
+func sub(from interface{}, to interface{}) (out int64, err error) {
```

Comment:
I would prefer to see that we either throw an error if `from` is less then `to`, or return a negative number.